powerline/segment.go

```diff
@@ -124,7 +146,11 @@ func getGitInformation() (string, bool) {
 	return status, staged
 }
 
-func GitSegment(t Theme) Segment {
+func GitSegment(cwdParts []string, t Theme) Segment {
+	if !isGitDir(cwdParts) {
```

Comment:
I think this should be done in `getGitInformation`, also isn't there a way to check same thing without constructing a path from it's parts?